docs/website/package.json

```diff
@@ -16,7 +16,7 @@
   "dependencies": {
     "@algolia/client-search": "^4.15.0",
     "@docusaurus/core": "^2.3.1",
-    "@docusaurus/preset-classic": "^2.3.1",
+    "@docusaurus/preset-classic": "^2.4.0",
```

Comment:
As with all bumps of any Docusaurus package, we'll need to update all `@docusaurus/*` dependencies to the same version (`2.4.0` here).